### DISCUSSION TITLE
ByteStash v1.5.3

### DISCUSSION THREAD

**Discussion by jordan-dalby** [jordan-dalby/ByteStash#140] [Announcements]
## Key Changes:
- Ability to disable accounts altogether, in this mode users will be auto-authenticated as the same user, meaning all snippets are shared.
- Ability to disable the standard login. When enabled, the standard login is removed, meaning that only OIDC users can log in.
- Fixes a bug with the issuer URL that caused some OIDC providers to report errors on sign in attempts
- Some great CI improvements from @stumpylog 
- Added a light theme
- The snippet modal and edit modal can now be horizontally expanded, great if you're storing larger snippets
- Improvements to the UX of the private view toggle

**Full Changelog**: https://github.com/jordan-dalby/ByteStash/compare/v1.5.2...v1.5.3

<hr /><em>This discussion was created from the release <a href='https://github.com/jordan-dalby/ByteStash/releases/tag/v1.5.3'>ByteStash v1.5.3</a>.</em>

**Comment by akalderon** [comment]
Look slike if I transition from the authenticated mode (v1.5.2) to an unauthenticated mode in v1.5.3 (DISABLE_ACCOUNTS=true and DISABLE_INTERNAL_ACCOUNTS=true), you are not associating the snippets of the original admin account to the anonymous account.
In other words, I lose access to all of my snippets and have to start all over again in disabled account mode.  From what I gather, you are creating an anonymous user account = 0 whereas the previous signed-in account was = 1.
Should it not transition smoothly retaining user id of 1 in non-authenticated mode and just disabling the authentication portion?

**Reply by akalderon** [reply]
Also can't go back to authenticated mode in v1.5.3 - setting the two flags to false does not bring back credentials prompt.  Have to revert to v1.5.2 to be prompted to login and regain access to my snippets.

**Reply by jordan-dalby** [reply]
@akalderon 
This is somewhat intended behaviour. The anonymous account is just that, a random, anonymous account, not the admin account. It sort of had to be designed like this for backwards compatibility with existing databases. Admin controls are really only necessary when multiple users are present. (And for right now, the admin actually can't do anything that a normal user could). If you want to transfer the snippets:
- Settings > Export
- Sign into admin account
- Settings > Import > Select file


> Also can't go back to authenticated mode in v1.5.3 - setting the two flags to false does not bring back credentials prompt. Have to revert to v1.5.2 to be prompted to login and regain access to my snippets.

This is odd. Have you definitely removed the DISABLE_INTERNAL_ACCOUNTS=true variable? This is the only value that controls whether the sign in form shows up or not.

**Reply by akalderon** [reply]
I didn't remove it.  I set it to false.
